src/components/Sidebar/index.jsx

```diff
@@ -1,107 +1,188 @@
-import { HomeOutlined, LabelOutlined, ArchiveOutlined, DeleteOutlineOutlined, AccountCircleOutlined, Logout, Close } from '@mui/icons-material/';
-
-
-import {v4 as uuid} from 'uuid';
-import { Link, useNavigate } from 'react-router-dom';
-
-import './sidebar.css';
-import { useAuth, useNotes } from 'contexts/';
-
+import {
+	HomeOutlined,
+	LabelOutlined,
+	ArchiveOutlined,
+	DeleteOutlineOutlined,
+	AccountCircleOutlined,
+	Logout,
+	Close,
+	Add,
+} from "@mui/icons-material/";
+
+import { v4 as uuid } from "uuid";
+import { Link, useNavigate } from "react-router-dom";
+
+import "./sidebar.css";
+import { useAuth, useNotes } from "contexts/";
+import { useState } from "react";
 
 const Sidebar = () => {
-
-    const navigate = useNavigate();
-    const { authDispatch, authUser, isAuth } = useAuth();
-    const { notesDispatch } = useNotes();
-
-    const sidebarSections = [
-        {
-            key: uuid(),
-            icon: <HomeOutlined />,
-            name: 'Home',
-            path: '/'
-        },
-        {
-            key: uuid(),
-            icon: <LabelOutlined />,
-            name: 'Labels',
-            path: '/labels'
-        },
-        {
-            key: uuid(),
-            icon: <ArchiveOutlined />,
-            name: 'Archive',
-            path: '/archive'
-        },
-        {
-            key: uuid(),
-            icon: <DeleteOutlineOutlined />,
-            name: 'Trash',
-            path: 'trash'
-        },
-        {
-            key: uuid(),
-            icon: <AccountCircleOutlined />,
-            name: 'Profile',
-            path: 'profile'
-        },
-    ];
-
-    const { showSidebar, handleShowSidebar } = useNotes();
-
-    const mappedSections = sidebarSections.map(({ key, icon, name, path}) => (
-        <h6 key={key} className="aside-link-wrapper">
-            <Link to={path} className="btn btn-link">
-                <span className="icon mui-icon aside-section-icon">
-                    {icon}
-                </span>
-                {name}
-            </Link>
-        </h6>
-    ));
-
-
-    const handleCreateNote = () => {
-        notesDispatch({ 
-            action: {
-                type: 'SHOW_NEW_NOTE_FORM',
-                payload: { showNewNoteForm: true, isEditing: null, editingNoteId: -1}
-            }
-        })
-    }
-
-    const handleLogout = () => {
-        localStorage.removeItem("inscribe-token");
+	const [newLabel, setNewLabel] = useState("");
+	const navigate = useNavigate();
+	const { authDispatch, authUser, isAuth } = useAuth();
+	const { notesDispatch, showSidebar, handleShowSidebar, labels } =
+		useNotes();
+
+	const sidebarSections = [
```

Comment:
i would suggest make a separate file for json data

Reply:
Yes, I have done this.